### ISSUE TITLE
Menu doesn't appear for my own module on 0.4.0 version

### ISSUE THREAD

**Issue by HoRaMusic** [VCVRack/Rack#245]
OS (if applicable): win7
Version (or "dev" if compiling from source): 0.4.0

When I use my own modules in the 0.4.1 version no menu appears when I right clic on the modules (but it works in dev version). I suspected that something needs to be changed in HPP or main CPP files of my modules series but they seem to be OK comparing to those released by VCV. Thanks

**Comment by AndrewBelt** [comment]
There is no 0.4.1 version. What are you taking about?

**Comment by rafzael** [comment]
Ok, I mean 0.4.0 sorry for the mistake

**Comment by AndrewBelt** [comment]
That probably means they're not loading. What does the terminal say when you launch Rack from the terminal? Where are your plugins and the plugin.dll files located?

**Comment by rafzael** [comment]
They dll is located like this: rack/plugins/treasure/treasure.dll
I retest when launched from the terminal and noticed that the menu appears sometime when I clik many times or when I move the move somewhere else. I also noticed than if I duplicate the module it crashes the rack.
I don't know if it's the same problem but I get a window message of json parsing error when I open the rack for the first time with the plugins added, and also when I open rack for the first time with the plugins deleted from the rack plugins directory. I had tested with a complied version of my plugin witheout the module that uses the json functions but the message still appears. 
Here is a sreenchot of the message
https://snag.gy/qNUaoQ.jpg

I verifed that all works perfectly in dev version and it's the case, it load the menu each time I clic and doesn't crash when I duplicate the modules.
